packages/whenable/README.md

```diff
@@ -0,0 +1,55 @@
+# Whenables
+
+Native promises are not compatible with `@agoric/store`, which means that on the Agoric platform, such promises are disconnected when their creator vat is upgraded.  Whenables are storable objects that represent a promise that tolerates disconnections.
```

Comment:
It isn't exactly @agoric/store that promises conflict with. They work fine with heap stores. They cannot be keys, but they are fine as mapStore or weakMapStore values. Therefore, also ok with heap zones. 

Even ok with virtual stores and virtual stores as values. But not even then, not stored virtually, and so not high cardinality. But since virtual is only per-incarnation, this is only a cardinality restriction, and not a semantic non-uniformity.

The conflict is only with durable stores, and is reflected in the behavior of `isStorable` on durable zones, which rejects promises. `isStorable` on heap zones and virtual zones just aliases `isPassable`, so passable promises are fine there.

Reply:
On rereading, I see that the compat point you're making here is not about durability of the original promise-like itself. It is about reconnection of a promise-like import of that original promise-like, after the vat containing the original durable promise-like is upgraded. But this still isn't an @agoric/store issue, yes?

Reply:
I tried to express things better now.

Reply:
You have. Resolving.